.gitignore

```diff
@@ -29,3 +30,25 @@ dist-gem/
 # Ignore gulp-generated vendor files
 #
 src/stylesheets/lib
+docs/assets/fonts/
+docs/assets/js/vendor/uswds-0.9.0.js
+docs/_scss/
+docs/assets/img/alerts/
+docs/assets/img/arrow-down.png
```

Comment:
Why do we need to ignore the exact file names? This seems very hard to maintain when we add new image files.